### DISCUSSION TITLE
Bootstrap classes work great on canvas but not in the style manager

### DISCUSSION THREAD

**Discussion by saudAtIrisdame** [GrapesJS/grapesjs#4089] [Q&A]
Bootstrap classes work great on canvas but not in the style manager.

I set the bg-danger background color class for heading. styles applied in canvas but not in style manager.
see the below image 👇 .
![image 4](https://user-images.githubusercontent.com/81280927/150540692-624ab958-982e-4ab9-a0aa-4e4b52e898d8.png)

@artf

**Comment by artf** [comment]
The StyleManager sees the CSS rules imported in [CssComposer](https://grapesjs.com/docs/api/css_composer.html) module, so if you're simply using the `config.canvas.styles` option, those style rules are not imported.

**Reply by saudAtIrisdame** [reply]
@artf  If I import bootstrap in CssComposer then it slows down editor loading

**Reply by artf** [reply]
Well yeah sure, that is expected, but show some metrics and the example of how you import it.
